src/commands/instances/group/GroupAchievementsCommand.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  bold,
+  Command,
+  CommandConfig,
+  CommandError,
+  formatDate,
+  getEmoji,
+  getLinkedGroupId
+} from '../../../utils';
+import { CommandInteraction, MessageEmbed } from 'discord.js';
+import womClient from '../../../services/wiseoldman';
+import config from '../../../config';
+
+const CONFIG: CommandConfig = {
+  name: 'achievements',
+  description: "View a group's most recent achievements."
+};
+
+class GroupAchievements extends Command {
+  constructor() {
+    super(CONFIG);
+  }
+
+  async execute(interaction: CommandInteraction) {
+    const groupId = await getLinkedGroupId(interaction);
+
+    try {
+      const groupAchievements = await womClient.groups.getGroupAchievements(groupId, { limit: 10 });
+
+      const achievementList = groupAchievements
+        .map(
+          ach =>
+            `${formatDate(ach.createdAt, 'DD MMM')} | ${bold(ach.player.displayName)} ${getEmoji(
+              ach.metric
+            )} ${ach.name}`
+        )
+        .join('\n');
+      const response = new MessageEmbed()
+        .setColor(config.visuals.blue)
+        .setTitle('Recent Group Achievements')
+        .setDescription(achievementList)
+        .setURL(`https://wiseoldman.net/groups/${groupId}/achievements/`);
+
+      await interaction.editReply({ embeds: [response] });
+    } catch (e) {
+      throw new CommandError("Couldn't find that group.");
```

Comment:
Could you make that catch block a chain to the `getGroupAchievements` function instead of putting everything inside try/catch? similar to how `GroupRecordsCommand` does it with `getGroupDetails`

Just so that if any other non-fetch related error is thrown, it's catched and reported properly.

Everything else looks good 👍

Reply:
updated!